app/modules/listings/components/users_listings.svelte

```diff
@@ -27,10 +51,27 @@
       flash = err
     }
   }
-</script>
 
+  const fetchMore = async () => {
+    if (fetching || hasMore === false) return
+    fetching = true
+    const nextListings = await getNextListingsBatch(offset, paginationSize)
+    if (isNonEmptyArray(nextListings)) {
+      listings = [ ...listings, ...nextListings ]
+    }
```

Comment:
`getNextListingsBatch` doesn't return anything, so this is dead code
```suggestion
    await getNextListingsBatch(offset, paginationSize)
```